src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -37,6 +44,8 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   session.execute(createTable)
   session.execute(createMetatdataTable)
   session.execute(createConfigTable)
+  for (tagId <- 1 to maxTagId)
```

Comment:
This actually creates three materialized views which may cause quite some overhead on Cassandra's side - I am not familiar with the implementation, but given how cassandra works it may be quite a lot of memory/size and performance overhead.